examples/utils/example-webpack-integration/src/webpackSpecifiedService.ts

```diff
@@ -0,0 +1,15 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { DefinePlugin } from "webpack";
+
+/**
+ * Helper to create a DefinePlugin for specifying the driver service to use.
+ * @internal
+ */
+export const createExampleDriverServiceWebpackPlugin = (service: string): DefinePlugin =>
+	new DefinePlugin({
+		EXAMPLE_DRIVER_SERVICE: JSON.stringify(service),
```

Comment:
?
```suggestion
		EXAMPLE_DRIVER_SERVICE: service,
```

Reply:
Never mind, I see the plugin says this is how it needs to be specified so double quotes end up in the final string that gets used during replacement.